### PR TITLE
Return failure when encountering unknown lang code, fixes #3524

### LINKED ISSUE #3524: Crash when using incorrect language setting in config.ini
**OS:** Windows 10
**Version:** 0.0.5-develop
**Commit:** fc74e50

- [ ] Reproducible in RCT2 (vanilla)?
- [ ] Multiplayer?

After update (I don't know what commit exactly), OpenRCT2 crashes just after run.
I solved this issue by changing config.ini file like this:
``language = 0`` -> ``language = ko-KR``

I think it should have a compatibility for old settings.

**Dump file**
[af2f40e0-3bc8-4951-a0fd-9012fabc9ba0.zip](https://github.com/OpenRCT2/OpenRCT2/files/253827/af2f40e0-3bc8-4951-a0fd-9012fabc9ba0.zip)

### PR DESCRIPTION
**Pull request by Gymnasiast**
Thanks to @YJSoft for pointing out the error.
Also changed language_open to a bool.

### CODE REVIEW

**Comment by janisozaur** [comment]
Can you please provide descriptive name for a commit?

File: src/localisation/language.cpp [hunk 1]
```diff
@@ -148,15 +148,15 @@ const char *language_get_string(rct_string_id id)
...
+bool language_open(int id)
 {
 	static const char *languagePath = "%s/language/%s.txt";
 	char filename[MAX_PATH];
 	char dataPath[MAX_PATH];
 
 	language_close_all();
 	if (id == LANGUAGE_UNDEFINED)
-		return 1;
+		return 0;
```
Comment (IntelOrca) [inline src/localisation/language.cpp]: Use `false`

File: src/world/map.c [hunk 2]
```diff
@@ -1638,15 +1638,15 @@ static money32 map_set_land_height(int flags, int x, int y, int height, int styl
 		return MONEY32_UNDEFINED;
 	}
 
-	if (height > 62) {
+	if (height > 270) {
```
Comment (duncanspumpkin) [inline src/world/map.c]: Can you add a comment stating this is a change from vanilla. Also don't sneak changes in on a unrelated fix

Comment (Gymnasiast) [inline src/world/map.c]: This shouldn't even have been in this commit :(

**Comment by Gymnasiast** [comment]
Ok, fixed everything.

File: src/localisation/language.cpp [hunk 3]
```diff
@@ -191,14 +191,14 @@ int language_open(int id)
...
 		}
 
 		// Objects and their localized strings need to be refreshed
 		reset_loaded_objects();
 
-		return 1;
+		return true;
 	}
 
 	return 0;
```
Comment (janisozaur) [inline src/localisation/language.cpp]: how about this?

**Pull request merged**